packages/app-builder-lib/src/targets/LinuxTargetHelper.ts

```diff
@@ -85,6 +85,8 @@ export class LinuxTargetHelper {
         return version.replace(/-/g, "_")
       case "rpm":
         return version.replace(/-/g, "~")
+      case "deb":
+        return version.replace(/-/g, "~")
```

Comment:
I think this can be represented with:
```
      case "rpm":
      case "deb":
        return version.replace(/-/g, "~")
```

Reply:
You're right. I pushed the change now, thanks!